src/steps/02-EvaluationCriteria/MRR/MarketResearchEfforts.vue

```diff
@@ -741,6 +749,15 @@ export default class MarketResearchEfforts extends Mixins(SaveOnLeave) {
       this.catalogReviewResults = "";
       sectionsWithNoSelectedCount++;
     }
+    if (this.cspIsOnlySourceCapable === "YES" 
+    || this.reviewedCatalogs === "YES"
+    && this.selectedTechniquesUsed.length !== 0 ) 
+    {
+      //if user selects yes to either question, adds sys id
+      // "Review of JWCC Contracts and/or contractor's catalog"
+      // to techniques used 
+      this.techniquesUsed += "," + this.techniqueJWCCSysId;
+    }
```

Comment:
Before the value is added, check to make sure that it is NOT already added.
Consider if the user exited and reentered the package 4 times and kept either Q1 or Q2 as yes, then the same sys ID would be appended four times.